docs/src/rules/no-console.md

```diff
@@ -41,7 +41,7 @@ Examples of **correct** code for this rule:
 /* eslint no-console: "error" */
 
 // custom console
-Console.log("Hello world!");
+console.log("Hello world!");
```

Comment:
Thanks for the PR. However, this is intentional and not a typo. You can check the comment above which says `custom console`